rcsbsearchapi/search.py

```diff
@@ -514,6 +518,37 @@ def __init__(self,
         super().__init__(service=STRUCTMOTIF_SEARCH_SERVICE, params=params)
 
 
+class ChemSimilarityQuery(Terminal):
+    """Special case of Terminal for chemical similarity search queries"""
+    def __init__(self, value: Optional[str] = None,
+                 query_type: ChemSimType = "formula",
+                 match_subset: Optional[bool] = False,
+                 descriptor_type: Optional[SubsetDescriptorType] = None,
+                 match_type: Optional[ChemSimMatchType] = None):
+        """Guide for making queries with query type - descriptor:
+        left side of the dash: front end (website) options for Match Type
+        right side of the dash: corresponding options for Match Type using Python tool
+        Similar Ligands (Stereospecific) - "graph-relaxed-stereo"
+        Similar Ligands (including Stereoisomers) - "graph-relaxed"
+        Similar Ligands (Quick screen) - "fingerprint-similarity"
+        Substructure (Stereospecific) - "sub-struct-graph-relaxed-stereo"
+        Substructure (including Stereoisomers) - "sub-struct-graph-relaxed"
+        Exact match - "graph-exact"
+        """
```

Comment:
Very nice to add this! One quick suggestion:

```suggestion
        """Guide for "descriptor_type" options:
        +-------------------------------------------+-----------------------------------+
        | Match Type                                | descriptor_type                   |
        +===========================================+===================================+
          Similar Ligands (Stereospecific)          | "graph-relaxed-stereo"
          Similar Ligands (including Stereoisomers) | "graph-relaxed"
          Similar Ligands (Quick screen)            | "fingerprint-similarity"
          Substructure (Stereospecific)             | "sub-struct-graph-relaxed-stereo"
          Substructure (including Stereoisomers)    | "sub-struct-graph-relaxed"
          Exact match                               | "graph-exact"
        +-------------------------------------------+-----------------------------------+
        """
```

Reply:
Got it! Just pushed the change.